packages/common/src/components/utils/InfoBox.tsx

```diff
@@ -28,9 +28,10 @@ interface Props {
   additionalWrapperStyles?: string
   squared?: boolean
   activateIconWrap?: boolean
+  isGlobal?: boolean
 }
 
-const InfoBox: React.FC<Props> = ({ type, children, additionalStyles, squared, additionalWrapperStyles, activateIconWrap }) => {
+const InfoBox: React.FC<Props> = ({ type, children, additionalStyles, squared, additionalWrapperStyles, activateIconWrap, isGlobal }) => {
```

Comment:
I feel like we´re trying to force too much functionality into this component, making the solution more complex. From a SOLID perspective it's violating the SRP principle. Will a `SystemBanner` ever need all the functionality of an `InfoBox`? Will any other notification component need the global property of an `InfoBox`? If not then perhaps the design would be better off with a specialized component. With React it's fairly cheap to create components and I think it would be beneficial to prefer several specialized, slightly similar, components over fewer god components with lots of configuration

Reply:
Most of the props sent to the component refer to styling. This is also the case for the newly added global property. Since the property refers to styling and not functionality I don't agree that it violates SRP. The isGlobal property determines if the info box should have the UX-framework defined global styling. Since system banners are specific to the notifications added from the IA application there could be other banners which also have the global styling. By creating another InfoBox component for the global look we would create duplicated code.

Reply:
The responsibility lies with determining which class to apply based on properties. Consider the alternative; a two single purpose components, e.g. `<InfoBox />` and `<GlobalInfoBox />`. In fact, since the properties refers to styling I would argue that the duplicated code will be minimal and trivial. It's already abstracted to CSS classes.

Reply:
To clarify, splitting the component into `<InfoBox />` and `<GlobalInfoBox />` was just an example, it's not the solution I'm suggesting.